src/main/kotlin/racingcar/PositionPrinter.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar
+
+import java.io.OutputStream
+import java.io.PrintWriter
+
+class PositionPrinter(private val positions: List<Positions>, outputStream: OutputStream) {
+    private val printWriter = PrintWriter(outputStream, true)
+
+    constructor(positions: List<Positions>) : this(positions, System.out)
+
+    fun print() {
+        for (positions in positions) {
+            positions.map { "-".repeat(it.intValue()) }
+                .forEach(printWriter::println)
+
+            printWriter.println()
+        }
```

Comment:
어떤 변수 이름이 다른 개발자도 쉽게 이해할 수 있는 변수 이름일까요? 아래의 코드로 개선해 보면 어떨까요?

```suggestion
        positions.forEach {
            it.map { position -> "-".repeat(position.intValue()) }
                .forEach(printWriter::println)

            printWriter.println()
        }
```

Reply:
변수이름 충돌이 있었군요. 컴파일 에러가 나지 않아서 놓친것 같습니다. (이럴땐 과하게 똑똑한..)
1168c7912683ba893427b4a19cb59867d7419a3f 에서 수정하였습니다.